distribution/lib/Standard/Base/0.0.0-dev/src/Nothing.enso

```diff
@@ -27,3 +29,14 @@ type Nothing
              "Hello".if_nothing ""
     if_nothing : Any -> Any
     if_nothing self ~function = function
+
+    ## Get a value for the key of the object.
+       As Nothing has no keys, returns `other`.
+
+       Arguments:
+       - key: The key to get.
+       - if_missing: The value to return if the key is not found.
+    get : Text | Integer -> Any -> Nothing
+    get self key ~if_missing=Nothing =
+        _ = [key]
```

Comment:
```suggestion
        _ = key
```

The vector seems unnecessary here. Although I hope it gets optimized-away anyway.